backend/server.js

```diff
@@ -27,7 +27,11 @@ app.use(express.urlencoded({ extended: true }))
 app.use(express.json())
 
 // CORS enabling middleware
-app.use(cors());
+app.use(cors({
+  origin: 'http://localhost:5173',
```

Comment:
Might be worth keeping the origin url in an environment variable, but that can be a later improvement depending on how you're hosting this (I have forgotten!)